client/src/pages/Inbox.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+// import CreateIcon from '@mui/icons-material/Create';
```

Comment:
Be sure to remove these commented-out sections unless you expect to use them in the coming days.